Source/RenderStream/Private/RenderStreamViewportClient.cpp

```diff
@@ -455,11 +683,8 @@ void URenderStreamViewportClient::Draw(FViewport* InViewport, FCanvas* SceneCanv
 
 					View->CameraConstrainedViewRect = View->UnscaledViewRect;
 
-					// Enable per-view virtual shadow map caching
-					View->State->AddVirtualShadowMapCache(MyWorld->Scene);
-
 					// Enable per-view Lumen scene
-					if (DisplayClusterLumenPerView->GetInt())
+					if (GDisplayClusterLumenPerView)
```

Comment:
This change is bad, you added the globals at the top so now 2 of them will be defined, we should be accessing them through their API as was done before.